### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by zeidlos** [twpayne/chezmoi#3422]
## Describe the bug

If I set up Encryption as per documentation, I get an age-error after chezmoi init.
`age: error: no identity matched any of the recipients`
(See: https://www.chezmoi.io/user-guide/frequently-asked-questions/encryption/#how-do-i-configure-chezmoi-to-encrypt-files-but-only-request-a-passphrase-the-first-time-chezmoi-init-is-run) 
Even after my documentation patch for `run_once_before_decrypt-private-key.sh`

I made sure that the public key in .chezmoi.toml.tmpl and ~/.config/chezmoi/chezmoi.toml  match. Also the key is present in ~/.config/chezmoi and the public key matches the other files.

## To reproduce

Follow the guide (https://www.chezmoi.io/user-guide/frequently-asked-questions/encryption/#how-do-i-configure-chezmoi-to-encrypt-files-but-only-request-a-passphrase-the-first-time-chezmoi-init-is-run) and run `chezmoi init --apply`.

## Expected behavior

It should work? ;)

## Output of command with the `--verbose` flag

```console
I didn't get any output using the verbose flag
```

## Output of `chezmoi doctor`

```console
RESULT    CHECK                       MESSAGE
ok        version                     v2.42.3, commit 43a4f5790ef1b709de881d36afb0a61745efa34f, built at 2023-12-16T23:08:59Z, built by Homebrew
ok        latest-version              v2.42.3
ok        os-arch                     darwin/arm64
ok        uname                       Darwin DELT-LMVT9Y1JW2 23.2.0 Darwin Kernel Version 23.2.0: Wed Nov 15 21:55:06 PST 2023; root:xnu-10002.61.3~2/RELEASE_ARM64_T6020 arm64
ok        go-version                  go1.21.5 (gc)
ok        executable                  /opt/homebrew/bin/chezmoi
ok        upgrade-method              brew-upgrade
ok        config-file                 ~/.config/chezmoi/chezmoi.toml, last modified 2023-12-21T00:03:50+01:00
warning   source-dir                  ~/.local/share/chezmoi is a git working tree (dirty)
ok        suspicious-entries          no suspicious entries
warning   working-tree                ~/.local/share/chezmoi is a git working tree (dirty)
ok        dest-dir                    ~ is a directory
ok        umask                       022
ok        cd-command                  found /bin/zsh
ok        cd-args                     /bin/zsh
info      diff-command                not set
ok        edit-command                found /usr/bin/vi
ok        edit-args                   /usr/bin/vi
ok        git-command                 found /opt/homebrew/bin/git, version 2.43.0
ok        merge-command               found /usr/bin/vimdiff
ok        shell-command               found /bin/zsh
ok        shell-args                  /bin/zsh
ok        age-command                 found /opt/homebrew/bin/age, version 1.1.1
info      gpg-command                 gpg not found in $PATH
info      pinentry-command            not set
info      1password-command           op not found in $PATH
info      bitwarden-command           bw not found in $PATH
info      bitwarden-secrets-command   bws not found in $PATH
info      dashlane-command            dcli not found in $PATH
info      doppler-command             doppler not found in $PATH
info      gopass-command              gopass not found in $PATH
info      keepassxc-command           keepassxc-cli not found in $PATH
info      keepassxc-db                not set
info      keeper-command              keeper not found in $PATH
info      lastpass-command            lpass not found in $PATH
info      pass-command                pass not found in $PATH
info      passhole-command            ph not found in $PATH
info      rbw-command                 rbw not found in $PATH
info      vault-command               vault not found in $PATH
info      vlt-command                 vlt not found in $PATH
info      secret-command              not set

```

</details>

**Comment by twpayne** [comment]
The commands work for me on a similar macOS machine. Nobody else has reported such a problem.

Are you sure you followed the instructions exactly? What are the exact commands that you ran? What happens if you follow the instructions again?

**Comment by twpayne** [comment]
Note that if you followed the instructions exactly you would have seen an error:

```
$ chezmoi init --apply
chmod: /Users/<name>/key.txt: No such file or directory
chezmoi: exit status 1
```

which was fixed in #3413 but not released yet.

**Comment by twpayne** [comment]
Closing due to inactivity. Please re-open if needed.
